### DISCUSSION TITLE
Error updating data through cipher

### DISCUSSION THREAD

**Discussion by maksimyugai** [zeapo/Android-Password-Store#1448] [Q&A] [answered]
Hi, guys. 

Faced with an error with the app while trying to pull changes from git new data. 

A full version of the error: "An error occurred during a Git operation. Error updating data through cipher."

I couldn't find where's app's log are storing on a mobile phone. It worked properly until some day.

What's it can be?

**Comment by msfjarvis** [comment]
The app's logs are not written persistently to the disk. If you know how to use [`logcat`](https://developer.android.com/studio/command-line/logcat) then you can try to record a log while attempting a connection.

**Comment by maksimyugai** [comment, marked as answer]
Solved problem by reinstalling.
